### PR TITLE
tweak data structure

### PR DESCRIPTION
**Pull request by halocline**

### CODE REVIEW

File: aries-site/src/data/structure.js [hunk 1]
```diff
@@ -69,51 +85,61 @@ export const structure = [
       'Tutorials, case studies, videos, how-to&aposs, articles, upcoming events and more about HPE Design and the Design System from the team.',
     icon: size => <IconResources size={size} />,
     pages: ['Examples', 'Videos', 'Designer', 'Themer', 'Grommet'],
+    parent: 'Home',
   },
   {
     name: 'Principles',
     description: 'Something',
     icon: '',
+    parent: 'Guidelines',
```
Comment (taysea) [inline aries-site/src/data/structure.js]: Is it redundant to have 'parent' on each of these pages and to have them listed as children in the above pages arrays? Just wondering if we plan to use 'parent' how every page or if it's only needed on the Home ones.

Comment (halocline) [reply aries-site/src/data/structure.js]: It is redundant, but would make for more efficient queries in certain cases. However, I thought of another approach which will still be efficient, but without the redundant data.

Comment (taysea) [reply aries-site/src/data/structure.js]: Cool, I'd love to chat about that.

Comment (halocline) [reply aries-site/src/data/structure.js]: Modified implementation. Check it out now.

File: aries-site/src/pages/index.js [hunk 2]
```diff
@@ -23,7 +23,11 @@ const HomeTiles = ({ ...rest }) => {
 };
 
 const title = 'Home';
-const topicList = structure.filter(page => page.parent === title);
+
+// Get details for each child page of the home page
+const topicList = structure
+  .find(page => page.name === title)
+  .pages.map(topic => structure.find(page => page.name === topic));
```
Comment (taysea) [inline aries-site/src/pages/index.js]: Nice!

**Pull request merged**